tests/test_move.py

```diff
@@ -60,3 +88,88 @@ def test_move_dir(self):
         if self.preserve_time:
             dst_file2_info = dst_fs.getinfo("bar/baz.txt", namespaces)
             self.assertEqual(dst_file2_info.modified, src_file2_info.modified)
+
+
+class TestMove(unittest.TestCase):
+    def test_move_file_tempfs(self):
+        with open_fs("temp://") as src, open_fs("temp://") as dst:
+            src_dir = src.makedir("Some subfolder")
+            src_dir.writetext("file.txt", "Content")
+            dst_dir = dst.makedir("dest dir")
+            fs.move.move_file(src_dir, "file.txt", dst_dir, "target.txt")
+            self.assertEqual(dst.readtext("dest dir/target.txt"), "Content")
+            self.assertFalse(src.exists("Some subfolder/file.txt"))
+
+    def test_move_file_fs_urls(self):
+        # create a temp dir to work on
+        with open_fs("temp://") as tmp:
+            path = tmp.getsyspath("/")
+            subdir_src = tmp.makedir("subdir_src")
+            subdir_src.writetext("file.txt", "Content")
```

Comment:
_Perhaps_ it's worth changing these two lines to:
```python
tmp.makedir("subdir_src")
tmp.writetext("subdir_src/file.txt", "Content")
```
and then change line 116 to
```python
self.assertFalse(tmp.exists("subdir_src/file.txt"))
```